conda/recipes/dask-cudf/meta.yaml

```diff
@@ -13,12 +13,12 @@ build:
 requirements:
   build:
```

Comment:
Unrelated to this PR, but it would be good to change this to `host`. Happy to pick that up if you don't want to handle it here.